### ISSUE TITLE
Support multiple Google Analytics IDs

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/WSUWP-Plugin-Analytics#15]
Once #13 is in place, we'll have flexibility in the number of GA IDs we can track. It would be nice to allow for:

* Network level trackers assigned to every site on a network.
* A number of trackers to be assigned on individual sites.

An interface should go with this that's nicer than a single text box. It may also be time to move the GA setting from Settings -> General at that point.

**Comment by jeremyBass** [comment]
completed

**Comment by jeremyfelt** [comment]
We have multiple trackers statically defined, but I want to leave this open to track a possible long term enhancement where multiple arbitrary trackers can be added to track analytics at the network level or to allow for multiple parties to add their tracker to an individual site. It is low priority though. :)

**Comment by jeremyfelt** [comment]
Related: #27

**Comment by jeremyBass** [comment]
oh yes I forgot that was the plan, when I read this I was thinking just about the js plugin, brain fart
